### ISSUE TITLE
word selection issue

### ISSUE THREAD

**Issue by singhranakumar22gmailcom** [readium/r2-navigator-swift#187]
i am useing readium library for epub .when i am holding on the first word then it's getting selected and when i am touching and holding on second word then it's get selected with previous one ,like whole sentence get's selected.

word selection issue

* `r2-shared-swift`:
* `r2-streamer-swift`:
* `r2-navigator-swift`:
* `r2-opds-swift`:
* `r2-lcp-swift`:

macOS: 11.2
platform: x86_64
Xcode 12.4
Build version 12D4e

iOS version: 14.4
Model (e.g. iPhone 11 Pro Max): iPhone 12 Pro Max
Is it an emulator? Yes

in utility.js

```
 document.addEventListener('selectionchange', debounce(50, function() {
        var info = {}
        var selection = document.getSelection();
       if (selection && selection.rangeCount > 0) {
            var rect = selection.getRangeAt(0).getBoundingClientRect();
            info['text'] = selection.toString().trim();
            info['frame'] = {
                'x': rect.left,
                'y': rect.top,
                'width': rect.width,
                'height': rect.height
            };
        }

        webkit.messageHandlers.selectionChanged.postMessage(info);
    }));
```

**Comment by mickael-menu** [comment]
I'm not sure what's going on, could you record a screencast showing the problem to lift any ambiguity?

We don't modify the web view selection in Readium.

https://user-images.githubusercontent.com/58686775/124120993-a9841800-da74-11eb-8267-55d3c28b32f6.mov

**Comment by singhranakumar22gmailcom** [comment]
https://user-images.githubusercontent.com/58514077/124142931-4ad89180-daa8-11eb-9f5c-4248c97da0cf.mov

in real device also it's  working same.

**Comment by mickael-menu** [comment]
Yeah it looks really buggy but it seems pretty specific to this book. Can you reproduce the problem with one of the books in the sample test app?

**Comment by singhranakumar22gmailcom** [comment]
Hii , can u please share the link of your sample test app.

**Comment by mickael-menu** [comment]
https://github.com/readium/r2-testapp-swift/tree/develop/Sources/Resources/Samples

**Comment by singhranakumar22gmailcom** [comment]
These are epub files, can u please share the simple Xcode project.

**Comment by mickael-menu** [comment]
It's in the same repo. But I asked to see if you had the same problem in your app with one of these sample books.

**Comment by ktech643** [comment]
@mickael-menu can you please guide me how to disable copy feature in Epub reading. I for the library and try to change that but app will crash with fork version on thread9 can you please guide me about that.
